com.unity.shadergraph/Documentation~/Block-Node.md

```diff
@@ -0,0 +1,34 @@
+# Block Node
+
+## Description
+
+A Block is a specific type of Node for the Master Stack. A block represents a single piece of the surface (or vertex) description data used in the final shader output. Some Block nodes are always available, while some may only be available with certain pipelines.  
+
+Certain blocks are only compatible with specific Graph Settings. Blocks may become Active or Inactive based on the graph settings. 
+
+Blocks cannot be cut, copied, or pasted. 
+
+## Adding and Removing Block Nodes
+
+New Block nodes can be added to a Context in the Master Stack by hovering over empty space in the Context and pressing space bar or right clicking and selecting “Create Node”. This will bring up the Create Node menu. 
+
+This menu will contain only Block nodes valid for that context. Vertex blocks will not appear in the Create Node menu of the Fragment context. 
+
+Selecting a block node from this menu will add it to the context. 
+
+Selecting a block node in the context and pressing “Delete” or right clicking and selecting “Delete” will remove the block from the context. 
+
+### Automatically Add or Remove Blocks
+
+Blocks can also be added or removed from a context automatically based on the user’s Shader Graph Preferences. If Automatically Add or Remove Blocks is enabled, the required Block nodes for a certain Target or Material type will be added automatically. Any incompatible block nodes that have no connections and default values will be removed from the context automatically. 
```

Comment:
Can we format the Automatically Add or Remove Blocks just to make it clear that its a direct setting?